src/v/coproc/ntp_context.h

```diff
@@ -30,8 +30,8 @@ namespace coproc {
 /// subscribers to an input ntp
 struct ntp_context {
     struct offset_pair {
-        model::offset last_read{model::model_limits<model::offset>::min()};
-        model::offset last_acked{model::model_limits<model::offset>::min()};
+        model::offset last_read{};
```

Comment:
This change doesn't do anything. model::offset is *already* `model::offset::min` which is min of int64_t

can you either remove this change, or 2 make it a separate `cosmetic: *` commit

Reply:
i think you can simplify your logic if you set to 0

Reply:
@graphcareful is this resolved?

Reply:
@dotnwat i'm waiting for feedback on what to do. Michal believes i shouldn't initialize to offset 0, alex is saying otherwise.

Reply:
if an initial value of 0 doesn't express anything meaningful, then that shouldn't be its an initial value.